### ISSUE TITLE
With gpu_mem=128 problems wiht fonts, showing covers and general display bugs

### ISSUE THREAD

**Issue by arthurlutz** [digitalLumberjack/recalbox-os#338]
When setting gpu_mem=128, the scraping works but there a number of bugs that render the rest unusable : 

* some fonts are displayed as squares
* flickering on some menus
* some covers don't show, then returning to the item show correctly

Do you want a picture / screenshot of the fonts displayed as squares ?

**Comment by digitalLumberjack** [comment]
Hi, yes i would like to have pictures of that. You have a 256mb rpi a or b ?

**Comment by arthurlutz** [comment]
![Screenshot](https://cloud.githubusercontent.com/assets/2734207/6000666/81969ba6-aad8-11e4-899f-9c7a2f8551c9.png)

This bug has been reported on emulationstation but I don't think it has had enough attention : https://github.com/Aloshi/EmulationStation/issues/374 and https://github.com/Aloshi/EmulationStation/issues/172#issuecomment-72143580

Am having this too on RetroPie, I think this is a big problem since it's excluding all the people that have Pis version1 from these projects.

**Comment by digitalLumberjack** [comment]
Ok, how many systems are displayed ? Is it happening if you have only default systems ?

**Comment by arthurlutz** [comment]
what do you can systems ? number of emulators in menu (those with rom files) : 8

What do you call "default systems" ?

**Comment by digitalLumberjack** [comment]
Yes that is what i wanted to know. Default as in "when you install recalbox but do not add roms". 
If you try to show onmy one system (removing roms in your directories except one) does this still happen?

**Comment by arthurlutz** [comment]
sorry can't test that out, the pi is being used for something else right now, but I would imagine the problem doesn't appear since the scraper will not find things. 

My theory is there is a shortage of memory when loading images from the internet and so it fails to load the font and boom.

**Comment by customcarvin** [comment]
I ran into the same problem... You guessed it right, not enough memory. Emulation Station is a ram hog for what it is; the only way for me to fix this was to use a lightweight theme and force the pi to 720p. Otherwise, what you attached above would happen when running out of memory. :bulb:Retroboy, any chance to enable shutdown of some services in ES menu to conserve mem? e.g. disable samba when not needed, etc.:question:

I'm actually working on my own custom theme atm for this specific issue as I have helped a lot of people with 256mb pi's install Recalbox on them, and they too are running into memory issues. Once I'm done I'll post it if there is interest... nickdj has already posted a couple low mem themes, take a look around here, and the forums if interested and you should find them; regardless, I still wanted to create my own. Besides, some of the svg logos are just downright amateur and look like someone used the "trace image" feature in their vector software because upon examination there are freaking nodes everywhere! I went through all the svg files for the systems I use, and cleaned all of the stray nodes and optimized vectors (still doing it in fact, as the MasterSystem logo isn't even close to the real thing, so I'm recreating that from scratch). In my theme I also added to the FB Alpha logo because everyone always asks, "what does that 'F' logo thing mean?" ... Hopefully what I have will clarify things a little more for them:grey_question:

Anyway, I'm rambling now... Here are a couple quick snapshots of what I have so far. Again, if there is interest then I'll post when finished; otherwise, just my friends and I will be using it :smile: 

![prev_fba](https://cloud.githubusercontent.com/assets/13281102/9347191/6d4c67ea-45f4-11e5-9f4c-6d556610279d.jpg)

![prev_gba](https://cloud.githubusercontent.com/assets/13281102/9347192/6d5397ea-45f4-11e5-97c0-2922da95e3b6.jpg)

![prev_nes](https://cloud.githubusercontent.com/assets/13281102/9347193/6d644a7c-45f4-11e5-867f-126f9bc76db5.jpg)

**Comment by arthurlutz** [comment]
@customcarvin thanks for your input. Indeed some better SVGs and lightweight themes could solve part of the problem. 

I've submitted an issue on that subject to EmulationStation https://github.com/Aloshi/EmulationStation/issues/499
